SELMs/App_Start/MapperConfig.cs

```diff
@@ -7,142 +7,283 @@
+				cfg.CreateMap<EquipmentHandoverFormDetailDTO, Equipment_Handover_Form_Detail>().ReverseMap()
+				.ForAllOtherMembers(act => act.NullSubstitute(null));
+
+				//Configuring Equipment_Import_Application_Detail and EquipmentImportApplicationDetailDTO
+
+
+				//Configuring Equipment_Location_History and EquipmentLocationHistoryDTO
+				cfg.CreateMap<EquipmentLocationHistoryDTO, Equipment_Location_History>().ReverseMap();
+
+				//Configuring Equipment_Project_History and EquipmentProjectHistoryDTO
+				cfg.CreateMap<EquipmentProjectHistoryDTO, Equipment_Project_History>().ReverseMap();
+
+				//Configuring Member_Location_History and MemberLocationHistoryDTO
+				cfg.CreateMap<MemberLocationHistoryDTO, Member_Location_History>().ReverseMap();
+
+				//Configuring Member_Project_History and MemberProjectHistoryDTO
+				cfg.CreateMap<MemberProjectHistoryDTO, Member_Project_History>().ReverseMap();
+
+				//Configuring Member_Project_History and MemberProjectHistoryDTO
+				cfg.CreateMap<AttachmentDTO, Attachment>()
+				.ForMember(des => des.date, act => act.MapFrom(src => DateTime.Parse(src.date)))
+				.ForAllOtherMembers(act => act.NullSubstitute(null));
+
+				cfg.CreateMap<Attachment, AttachmentDTO>()
+				.ForMember(des => des.date, act => act.MapFrom(src => src.date.ToString()))
+				.ForAllOtherMembers(act => act.NullSubstitute(null));
+
+				//Configuring InverntoryRequestApplicationDTO and Inverntory_Request_Application
+				cfg.CreateMap<InventoryRequestApplicationDTO, Inventory_Request_Application>().ReverseMap()
+				.ForAllOtherMembers(act => act.NullSubstitute(null));
+
+				//Configuring InverntoryRequestApplicationDetailDTO and Inverntory_Request_Application_Detail
+				cfg.CreateMap<InventoryRequestApplicationDetailDTO, Inventory_Request_Application_Detail>()
+				.ForMember(des => des.inventory_date, act => act.MapFrom(src => DateTime.Parse(src.inventory_date)))
+				.ForAllOtherMembers(act => act.NullSubstitute(null));
+
+
+				cfg.CreateMap<Inventory_Request_Application_Detail, InventoryRequestApplicationDetailDTO>()
+				.ForMember(des => des.inventory_date, act => act.MapFrom(src => src.inventory_date.ToString()))
+				.ForAllOtherMembers(act => act.NullSubstitute(null));
+
+				//ConfiguringInverntory_Request_Application_Detail
+				cfg.CreateMap<Inventory_Request_Application_Detail, Inventory_Request_Application_Detail>()
+				.ForAllMembers(opt => opt.Condition((src, dest, srcMember) => srcMember != null));
+
+
+				//Configuring Equipment_Handover_Form and EquipmentHandoverFormDTO
+				cfg.CreateMap<EquipmentHandoverFormDTO, Equipment_Handover_Form>()
+				.ForMember(des => des.create_date, act => act.MapFrom(src => DateTime.Parse(src.create_date)))
+				.ForMember(des => des.handover_date, act => act.MapFrom(src => DateTime.Parse(src.handover_date)))
+				.ForMember(des => des.receipt_date, act => act.MapFrom(src => DateTime.Parse(src.receipt_date)))
```

Comment:
receipt_date ko tồn tại ?
'EquipmentHandoverFormDTO' does not contain a definition for 'receipt_date' and no accessible extension method 'receipt_date' accepting a first argument of type 'EquipmentHandoverFormDTO' could be found (are you missing a using directive or an assembly reference?)

![image](https://github.com/SEP490-G37-FA23/SELMs/assets/91528893/f7873938-ab7c-46f5-8e48-d5cd3a900700)